contrib/pkg/createcluster/create.go

```diff
@@ -154,31 +181,33 @@ func NewCreateClusterCommand() *cobra.Command {
 	flags.StringVar(&opt.PullSecret, "pull-secret", "", "Pull secret for cluster. Takes precedence over pull-secret-file.")
 	flags.StringVar(&opt.DeleteAfter, "delete-after", "", "Delete this cluster after the given duration. (i.e. 8h)")
 	flags.StringVar(&opt.PullSecretFile, "pull-secret-file", defaultPullSecretFile, "Pull secret file for cluster")
-	flags.StringVar(&opt.AWSCredsFile, "aws-creds-file", defaultAWSCredsFile, "AWS credentials file")
+	flags.StringVar(&opt.CredsFile, "creds-file", "", "Cloud credentials file")
```

Comment:
for the description, maybe it should be extended to something like: "Cloud credentials file (different defaults depending on selected cloud)"